book/visual-effects.md

```diff
@@ -1181,7 +1181,7 @@ conceptual `Pixel` class I introducd earlier.
 When we apply a `Paint` with an `Alphaf` parameter, the first thing
 Skia does is add the requested opacity to each pixel:
 
-``` {.python file=examples}
+``` {.python file=examples .example}
 class Pixel:
```

Comment:
This might be turning off the auto-comparison for these snippets (they're normally compared to, I think, `example11.py`. I think auto-compare skips things labeled `example`. I'm not against the change though. Maybe we need to tweak something in `infra/compare.py`.